sbnanaobj/StandardRecord/SRTrkLikelihoodPID.h

```diff
@@ -0,0 +1,29 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRTrkLikelihoodPID.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRTRKLIKELIHOODPID_H
+#define SRTRKLIKELIHOODPID_H
+
+namespace caf
+{
+  /// Track PID from dE/dx v. likelihood from dE/dx PDF measured with residual range-based kinetic energy and hit pitch
+  class SRTrkLikelihoodPID
+    {
+    public:
+
+      SRTrkLikelihoodPID();
+      virtual ~SRTrkLikelihoodPID();
```

Comment:
Remove the destructor, since it does not do anything ([`CF-151`](https://sbnsoftware.github.io/sbn/codingconv/CodingConventionsExplained.html#CF151)).
Also I would like to know if you copied that idea from anywhere in our code, so that we can fix that too.
```suggestion
```
_[edit]_ I guess that's `SRTrkChi2PID`.